### PR TITLE
new(config): add almalinux build-drivers jobs

### PR DESCRIPTION
**Pull request by jordyb6**
Added a job for building kmods/ebpf probes for almalinux, based on reading this conversation:
https://kubernetes.slack.com/archives/CMWH3EH32/p1677767734697179?thread_ts=1676538027.728449&cid=CMWH3EH32

### CODE REVIEW

**Comment by FedeDP** [comment]
Hi! Thanks for opening this PR! :) I am going to comment on it soon!

File: config/jobs/build-drivers/build-new-almalinux.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+presubmits:
+  falcosecurity/test-infra:
+  - name: validiate-new-drivers-debian-presubmit
```
Comment (FedeDP) [inline config/jobs/build-drivers/build-new-almalinux.yaml:3]: Name `debian` is wrong here :)

In whole file you need to substitute `debian` with `almalinux` .

Comment (jordyb6) [reply config/jobs/build-drivers/build-new-almalinux.yaml:3]: Oh wow, I could've sworn I did that, my bad!

Comment (FedeDP) [reply config/jobs/build-drivers/build-new-almalinux.yaml:3]: :D np!

Comment (jordyb6) [reply config/jobs/build-drivers/build-new-almalinux.yaml]: I pushed the changes now.

**Comment by FedeDP** [comment]
Mmmh can you rebase? I don't actually think there is any conflict though :/

**Comment by FedeDP** [comment]
Uh you need to `rebase` not `merge` :)

**Comment by jordyb6** [comment]
@FedeDP I'm sorry, I'm quite new to git and fussed up a little bit.
Is everything fine now? Since I see my commits are already integrated in master.

**Comment by FedeDP** [comment]
Wow i don't know what happened :laughing: 
@leogr any idea?
The PR is closed (not merged) but its commits were merged into master :thinking:  ...super weird!
I am sad because the PR status is closed and not merged though, but yep, the PR is actually merged!
Thanks for your contribution!

**Pull request closed without merging**